Mathlib/Data/PNat/Defs.lean

```diff
@@ -0,0 +1,294 @@
+/-
+Copyright (c) 2017 Microsoft Corporation. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Mario Carneiro, Neil Strickland
+-/
+
+import Mathlib.Algebra.NeZero
+import Mathlib.Order.Basic
+
+/-!
+# The positive natural numbers
+
+This file contains the definitions, and basic results.
+Most algebraic facts are deferred to `Data.PNat.Basic`, as they need more imports.
+-/
+
+
+/-- `ℕ+` is the type of positive natural numbers. It is defined as a subtype,
+  and the VM representation of `ℕ+` is the same as `ℕ` because the proof
+  is not stored. -/
+def PNat :=
+  { n : ℕ // 0 < n }
+  deriving DecidableEq, LinearOrder
+#align pnat PNat
+
+@[inherit_doc]
+notation "ℕ+" => PNat
+
+instance : One ℕ+ :=
+  ⟨⟨1, Nat.zero_lt_one⟩⟩
+
+instance coePNatNat : Coe ℕ+ ℕ :=
+  ⟨Subtype.val⟩
+#align coe_pnat_nat coePNatNat
+
+instance : Repr ℕ+ :=
+  ⟨fun n n' => reprPrec n.1 n'⟩
+
+namespace PNat
+
+-- Porting note: no `simp` due to eagerly elaborated coercions
+theorem mk_coe (n h) : ((⟨n, h⟩ : ℕ+) : ℕ) = n :=
+  rfl
+#align pnat.mk_coe PNat.mk_coe
+
+/-- Predecessor of a `ℕ+`, as a `ℕ`. -/
+def natPred (i : ℕ+) : ℕ :=
+  i - 1
+#align pnat.natPred PNat.natPred
```

Comment:
```suggestion
#align pnat.nat_pred PNat.natPred
```
And below. Could you double check all the #aligns, then mark as ready for review again?

Reply:
Doh. Thanks, done